src/v/utils/human_chrono.h

```diff
@@ -0,0 +1,86 @@
+/*
```

Comment:
nit: i think it would be fine to stick this in the existing human.h file to keep things simple.